packages/manager/apps/hub/src/components/hub-order-tracking/HubOrderTracking.component.tsx

```diff
@@ -192,15 +201,15 @@ export default function HubOrderTracking() {
                       color={ODS_THEME_COLOR_INTENT.primary}
                       className="inline-block mr-1"
                     >
-                      <strong>{format(new Date(currentStatus.date))}</strong>
-                      &nbsp;{t(`order_tracking_history_${currentStatus.label}`)}
+                      <strong>{format(new Date(displayedStatus.date))}</strong>
+                      &nbsp;{t(`order_tracking_history_${displayedStatus.label}`)}
```

Comment:
`disaplyedStatus` can be undefined
```suggestion
                      <strong>{format(new Date(displayedStatus?.date))}</strong>
                      &nbsp;{t(`order_tracking_history_${displayedStatus?.label}`)}
```